src/protocol/logpoint.ts

```diff
@@ -226,7 +226,7 @@ export async function setLogpoint(
   condition: string,
   showInConsole: boolean = true
 ) {
-  await ThreadFront.ensureAllSources();
+  await Promise.all([ThreadFront.ensureAllSources(), ThreadFront.ensureProcessed("basic")]);
```

Comment:
I'm *pretty* sure that just waiting for basic processing works here, but I could be wrong?

Reply:
not opposed, but isn't the backend equipped to handle analysis before basic processing is finished?

Reply:
Is this the right place to wait? There are other places that start analyses (e.g. `setEventLogpoint()`), so I think we should wait in `AnalysisManager.runAnalysis()` - see also #6701

Reply:
The problem is that it's not a lack of sources which is the issue here. The sources were loaded before the "wrong"  analysis was run in the case of https://app.replay.io/recording/invalid-no-hits--8c51609c-f563-4f53-8857-dba1005ee64a?point=546164725844869027945326080604256823&time=232105&hasFrames=false. But it's true that if we just fix the re-running, then this kind of error will be a lot less problematic, since the analysis will re-run when everything is loaded.